traffic_ops/app/db/reencrypt/reencrypt.go

```diff
@@ -0,0 +1,406 @@
+package main
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import (
+	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/trafficvault/backends/postgres"
```

Comment:
Instead of importing Traffic Ops internals, should the function you're using be pulled out into e.g. `lib/go-tc/`? Or some new library for crypto things like `lib/crypto/` maybe?

Reply:
ill update this next week. Steve also has a PR that will move them so ill work with him to make sure we get them in the same place to minimize rebasing headache

Reply:
done!